### PR TITLE
azure-nvme-utils: Migrate coreos-overlay to portage-stable, 0.2.0 bump

### PR DESCRIPTION
**Pull request by chewi**
# azure-nvme-utils: Migrate to portage-stable, 0.2.0 bump

The coreos-overlay package under app-admin was written by Jeremi around the same time I added it to Gentoo under sys-apps. It has had a new release since.

## How to use

Build the package?

## Testing done

A Jenkins packages_all_arches job was run against Azure. All passed and no notable changes.

- [X] Changelog entries added in the respective `changelog/` directory (user-facing change, bug fix, security fix, update)
- [X] Inspected CI output for image differences: `/boot` and `/usr` size, packages, list files for any missing binaries, kernel modules, config files, kernel modules, etc.

### CODE REVIEW

File: .github/workflows/portage-stable-packages-list [hunk 1]
```diff
@@ -546,6 +546,7 @@ sec-policy/selinux-unconfined
 
 sys-apps/acl
 sys-apps/attr
+sys-apps/azure-nvme-util
```
Comment (jepio) [inline .github/workflows/portage-stable-packages-list:549]:
```suggestion
sys-apps/azure-nvme-utils
```

**Comment by chewi** [comment]
Now properly testing on Azure using Kola, as requested.

```
core@kola-9129ebae-969a5c49bd ~ $ ls -l /dev/disk/azure/
total 0
drwxr-xr-x. 4 root root 80 Aug 26 13:02 local
lrwxrwxrwx. 1 root root 13 Aug 26 13:02 os -> ../../nvme0n1
lrwxrwxrwx. 1 root root 15 Aug 26 13:02 os-part1 -> ../../nvme0n1p1
lrwxrwxrwx. 1 root root 15 Aug 26 13:02 os-part2 -> ../../nvme0n1p2
lrwxrwxrwx. 1 root root 15 Aug 26 13:02 os-part3 -> ../../nvme0n1p3
lrwxrwxrwx. 1 root root 15 Aug 26 13:02 os-part4 -> ../../nvme0n1p4
lrwxrwxrwx. 1 root root 15 Aug 26 13:02 os-part6 -> ../../nvme0n1p6
lrwxrwxrwx. 1 root root 15 Aug 26 13:02 os-part7 -> ../../nvme0n1p7
lrwxrwxrwx. 1 root root 15 Aug 26 13:02 os-part9 -> ../../nvme0n1p9
```

**Comment by jepio** [comment]
> Now properly testing on Azure using Kola, as requested.
> 
> ```
> core@kola-9129ebae-969a5c49bd ~ $ ls -l /dev/disk/azure/
> total 0
> drwxr-xr-x. 4 root root 80 Aug 26 13:02 local
> lrwxrwxrwx. 1 root root 13 Aug 26 13:02 os -> ../../nvme0n1
> lrwxrwxrwx. 1 root root 15 Aug 26 13:02 os-part1 -> ../../nvme0n1p1
> lrwxrwxrwx. 1 root root 15 Aug 26 13:02 os-part2 -> ../../nvme0n1p2
> lrwxrwxrwx. 1 root root 15 Aug 26 13:02 os-part3 -> ../../nvme0n1p3
> lrwxrwxrwx. 1 root root 15 Aug 26 13:02 os-part4 -> ../../nvme0n1p4
> lrwxrwxrwx. 1 root root 15 Aug 26 13:02 os-part6 -> ../../nvme0n1p6
> lrwxrwxrwx. 1 root root 15 Aug 26 13:02 os-part7 -> ../../nvme0n1p7
> lrwxrwxrwx. 1 root root 15 Aug 26 13:02 os-part9 -> ../../nvme0n1p9
> ```

Does the `local` also have symlinks for the other nvme drive?

**Review by jepio — APPROVED** [review]
lets just validate the other symbolic links and then: good to go

**Comment by chewi** [comment]
I have this in my scrollback, good enough?

```
core@kola-9129ebae-969a5c49bd ~ $ ls -l /dev/disk/azure/local/by-name/
total 0
lrwxrwxrwx. 1 root root 19 Aug 26 13:02 nvme-220G-1 -> ../../../../nvme1n1
```

**Comment by jepio** [comment]
Yup, thanks

File: changelog/updates/2024-08-23-azure-nvme-utils-update.md [hunk 2]
```diff
@@ -0,0 +1 @@
+- azure-nmve-utils ([0.2.0](https://github.com/Azure/azure-nvme-utils/releases/tag/v0.2.0))
```
Comment (dongsupark) [inline changelog/updates/2024-08-23-azure-nvme-utils-update.md:1]: Nit: `s/nmve/nvme/`

I have just manually fixed in the release notes draft.

Comment (chewi) [reply changelog/updates/2024-08-23-azure-nvme-utils-update.md:1]: Woops, thanks. Ironically, the project just renamed itself to azure-vm-utils!

**Pull request merged**